capitalize_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    super.capitalize
+  end
+end
```

Comment:
- [ ] The `correct_name` method here should call the `capitalize` method from the chain of the `@nameable.correct_name` method as it's not defined in the super class.

Reply:
Suggestion implemented